pkg/cache/clusterqueue.go

```diff
@@ -141,6 +140,16 @@ func newCohort(name string, size int) *Cohort {
 	}
 }
 
+func (c *Cohort) Lendable() map[corev1.ResourceName]int64 {
+	lendable := make(map[corev1.ResourceName]int64)
+	for _, member := range c.Members.UnsortedList() {
```

Comment:
```suggestion
	for member := range c.Members {
```

This will avoid unnecessary allocations

Reply:
Thanks